packages/@sanity/cli/src/actions/init-project/initProject.ts

```diff
@@ -996,6 +1042,16 @@ export default async function initSanity(
       )
     }
 
+    if (
+      cliFlags.quickstart &&
+      (cliFlags.project || cliFlags.dataset || cliFlags.visibility || cliFlags.template)
+    ) {
+      const disallowed = ['project', 'dataset', 'visibility', 'template']
+      const usedDisallowed = disallowed.filter((flag) => cliFlags[flag as keyof InitFlags])
+      const usedDisallowedStr = usedDisallowed.map((f) => `--${f}`).join(', ')
```

Comment:
nit for consistency with the line above:
```suggestion
      const usedDisallowedStr = usedDisallowed.map((flag) => `--${flag}`).join(', ')
```